MAINTAINERS.rst

```diff
@@ -0,0 +1,26 @@
+Maintainers
```

Comment:
I think adding GitHub user names here would be useful.

Reply:
Maybe hyperlinks to gh profile pages (just thinking from an UX perspective).

Reply:
Done